Dalamud/Plugin/PluginLog.cs

```diff
@@ -12,38 +14,262 @@ public static class PluginLog
         /// <summary>
         /// Log a templated message to the in-game debug log.
         /// </summary>
+        /// <param name="level">The log level to emit.</param>
         /// <param name="messageTemplate">The message template.</param>
         /// <param name="values">Values to log.</param>
-        public static void Log(string messageTemplate, params object[] values)
+        private static void Emit(LogEventLevel level, string messageTemplate, params object[] values)
```

Comment:
Nit, but is it possible for this to direct to the version of the function with an Exception argument, with that argument null here? As in:
```csharp
private static void Emit(LogEventLevel level, string messageTemplate, params object[] values)
    => Emit(level, null, messageTemplate, values);
```
I'm not sure if Serilog provides an overload that allows that to be done cleanly, if not then this can be disregarded.

Reply:
Null exceptions are handled by serilog properly, as in they dont show up.

```
Log.Information(null as Exception, "There is no exception here.");
Log.Information(new Exception("howdy"), "There is an exception here.");
```
```
2020-10-11 18:30:41.732 -04:00 [INF] There is no exception here.
2020-10-11 18:30:41.732 -04:00 [INF] There is an exception here.
System.Exception: howdy
```